vllm/model_executor/models/llava_onevision.py

```diff
@@ -45,19 +45,20 @@
 
 # For profile run
 _MAX_FRAMES_PER_VIDEO = 16
-_MAX_NUM_VIDEOS = 1
+_MAX_NUM_VIDEOS = 8
 
 
 class LlavaOnevisionVideoPixelInputs(TypedDict):
     type: Literal["pixel_values_videos"]
     data: Union[torch.Tensor, List[torch.Tensor]]
     """
-    Shape: `(batch_size, num_frames, num_channels, height, width)`
+    Shape: `(batch_size, num_videos, num_frames, num_channels, height, width)`
 
-    Note that `num_frames` may be different for each batch, in which case
-    the data is passed as a list instead of a batched tensor.
+    Note that `num_videos` may be different for each batch, and 'num_frames'
+    may be different for each video, in which case the data is passed as a
+    list instead of a batched tensor.
 
-    Note that it only supports one video input for one batch.
+    Note that it only supports single batch.
```

Comment:
Does "single batch" here mean `batch_size` is limited to 1? If you intend to keep it this way in the PR, you should check for this and raise an error if the `batch_size` is found to be more than 1 when videos are being inputted.

Reply:
I only tested cases with single batch. How can I organize the multi batch inputs for multi model data？

Reply:
If you pass in multiple prompts to `llm.generate` then it's already multi-batch. Just need to make sure `max_num_seqs` is not limited to 1.

Reply:
Thank you for your response.  Setting prompt as a list triggers  a bug in `input_processor_for_clip`.

Reply:
This model can support single/multi batch with single or multi videos now.